### ISSUE TITLE
Adding a Router for the Example Site?

### ISSUE THREAD

**Issue by frastlin** [WhatSock/accdc-react#28]
AccDC needs a router for the example site.
Does AccDC have this functionality already?
I normally use [React Router,](https://reacttraining.com/react-router/web/guides/quick-start) but don't know what AccDC already has.
The router is used to make a single-page app have multiple pages and URLS.

**Comment by accdc** [comment]
Hi, not really. Only the Tab Module is used for inter-content traversal, so you shouldn't have any issues by adding this to the project.

**Comment by frastlin** [comment]
Page navigation is something we should consider adding as a component. making the first heading an h1, jumping to that h1 when the page loads, having a sidebar and a footer, and a logo are all things that would be useful to have.

**Comment by accdc** [comment]
That sounds good to me, and could be fairly easy to implement as a single React element to auto-generate internal page links. Unfortunately I'm bouncing around so many projects right now, I'm not sure when I could get to it, likely not til after the new year at the soonest. Is this something you want to put in sooner than that?

**Comment by frastlin** [comment]
I've actually already built a component for another project that does this, I just need to make it more extendable and put it into the components. I'll post the design a little later.

**Comment by accdc** [comment]
Awesome!

As an FYI, which you probably already know, the API $A.focus() method is useful in such cases.

**Comment by frastlin** [comment]
Yes, I need to play with $A.focus and a react component, because focusing with react is a bit of a pain and doesn't always work when you load lots of information each page switch. So currently in my page component, I'm focusing using a ref, but it works only half the time.
